swagger/v1/provider.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Provider",
+  "description": "Provider",
+  "id": "provider",
+  "stability": "prototype",
+  "strictProperties": true,
+  "type": "object",
+  "definitions": {
+    "id": {
+      "description": "unique identifier of the provider",
+      "example": "48c0e8f5-0522-4ecf-82ae-05363f235722",
+      "format": "uuid",
+      "type": "string"
+    },
+    "type": {
```

Comment:
This is a bit confusing. Is this `type` only ever an enum which can only be "provider"? If so, the example needs to be singular. But an enum would usually be a fixed set of strings that this could be. E.g. Jurisdiction which is   "MAGISTRATES" or "CROWN"